home/pi/auto_run.sh

```diff
@@ -380,7 +396,7 @@ function setup_wizard() {
 
                 # Flash latest Seeed firmware
                 echo "Downloading and flashing latest firmware from Seeed..."
-                sudo /home/pi/mycroft-core/.venv/bin/pip install pyusb click
+                sudo /home/pi/mycroft-core/.venv/bin/pip3 install pyusb click
```

Comment:
note that on systems with python2 AND python3, this might be needed .. BUT, afaict, since we should be within the picroft venv ... shouldn't technically be required; and anyway `pip` and `pip3` are the same on picroft venv systems
```
(.venv) pi@picroft:~ $ diff /home/pi/mycroft-core/.venv/bin/pip /home/pi/mycroft-core/.venv/bin/pip3ip3
(.venv) pi@picroft:~ $ 

(.venv) pi@picroft:~ $ which pip
/home/pi/mycroft-core/.venv/bin/pip

(.venv) pi@picroft:~ $ ll /home/pi/mycroft-core/.venv/bin/ | grep pip
-rwxr-xr-x 1 pi pi   243 Jun 20 02:19 pip
-rwxr-xr-x 1 pi pi   243 Jun 20 02:19 pip3
-rwxr-xr-x 1 pi pi   243 Jun 20 02:19 pip3.7
```